strippy.ps1

```diff
@@ -398,7 +409,7 @@ function output-keylist ($finalKeyList, $listOfSanitisedFiles) {
         }
         
         log outkey message "Exporting KeyList to $kf"
-        $KeyOutfile = (eval-config-string $script:config.KeyListFirstline) + "`r`n" + $( $finalKeyList | Out-String )
+        $KeyOutfile = (eval-config-string $script:config.KeyListFirstline) + "`r`n" + $( $finalKeyList.GetEnumerator() | sort -Property name | Out-String )
```

Comment:
Good solution. In my own branch I was spinning off a bunch of code. 

My only wish is that the sort function took into account `stringNumeral` sorting. It doesn't though and it's not important enough to add. It would just mean that Address9 comes after Address8 rather than Address89..